test/demo/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: demo-ingress
+spec:
+  rules:
+  - host:
```

Comment:
We don't want to limit the service using the `Host:` header so better move the `host` key.